### PR TITLE
fix: remove package json

### PR DESCRIPTION
**Pull request by tedim52**
## Description:
These files aren't need at the base of the monorepo and we're mistakenly added in previous commits.

## Is this change user facing?
NO

### CODE REVIEW

File: go.work.sum [hunk 1]
```diff
@@ -30,12 +30,15 @@ cloud.google.com/go/automl v1.13.1 h1:iP9iQurb0qbz+YOOMfKSEjhONA/WcoOIjt6/m+6pIg
 cloud.google.com/go/automl v1.13.1/go.mod h1:1aowgAHWYZU27MybSCFiukPO7xnyawv7pt3zK4bheQE=
```
Comment (h4ck3rk3y) [inline go.work.sum:30]: why is this here?

**Review by h4ck3rk3y — APPROVED** [review]
LGTM, is the go work intentional?

File: go.work.sum (hunk 1 above)
Comment (tedim52) [reply go.work.sum:30]: the `go.work` always gets changed with every change and I always commit it bc Go instructs you to but I don't think it has any effect.
